Makefile

```diff
@@ -22,11 +19,12 @@ build:
 	@echo Nothing to do.
 
 docker-build:
+	curl -Ls https://dl.bintray.com/deis/deisci/objstorage-0314fc5-linux-amd64 -O
+	mv objstorage-0314fc5-linux-amd64 rootfs/bin/objstorage
+	chmod +x rootfs/bin/objstorage
```

Comment:
@kmala repeating what I commented [here](https://github.com/deis/slugrunner/pull/26/files#r59243795), can you do these steps inside the `Dockerfile` instead of here?